client/package.json

```diff
@@ -38,18 +45,20 @@
   "devDependencies": {
     "autoprefixer": "6.5.3",
     "babel-core": "6.18.2",
-    "babel-eslint": "5.0.0",
+    "babel-eslint": "7.1.1",
     "babel-jest": "17.0.2",
     "babel-loader": "6.2.8",
     "babel-preset-es2015": "6.18.0",
     "babel-preset-react": "6.16.0",
     "clean-webpack-plugin": "0.1.14",
     "css-loader": "0.26.1",
-    "eslint": "2.4.0",
-    "eslint-config-airbnb": "6.1.0",
-    "eslint-loader": "1.3.0",
-    "eslint-plugin-jasmine": "1.6.0",
-    "eslint-plugin-react": "4.2.2",
+    "eslint": "3.11.1",
+    "eslint-config-airbnb": "13.0.0",
+    "eslint-loader": "1.6.1",
+    "eslint-plugin-import": "2.2.0",
+    "eslint-plugin-jasmine": "2.2.0",
```

Comment:
I thought jest tests use jasmine syntax, so you get new keywords like `describe` and friends that the linter needs to know about.